### PR TITLE
add workflow for aks database backup and restore

### PR DESCRIPTION
**Pull request by neillturner**
### Context

need to have daily backups and a working restore workflow after production migration of rsm to aks

### Changes proposed in this pull request

add workflows for aks database backup and restore.
until aks in production the scheduled database backup is commented out 
code based on latest templates from teachers services cloud repo 

### Guidance to review

see successful backup runs
https://github.com/DFE-Digital/refer-serious-misconduct/actions/runs/12067585770
https://github.com/DFE-Digital/refer-serious-misconduct/actions/runs/12067542493

### Link to Trello card

[rsm: add workflow for aks database backup and restore](https://trello.com/c/YC6xGcSC/2156-rsm-add-workflow-for-aks-database-backup-and-restore)

### Checklist

- [ ] Attach to Trello card
- [ ] Rebased main
- [ ] Cleaned commit history
- [ ] Tested by running locally

### CODE REVIEW

**Comment by neillturner** [comment]
[rsm: add workflow for aks database backup and restore](https://trello.com/c/YC6xGcSC/2156-rsm-add-workflow-for-aks-database-backup-and-restore)

File: .github/workflows/backup-db.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
+name: Backup database
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: Environment to backup
+        required: true
+        default: development
```
Comment (RMcVelia) [inline .github/workflows/backup-db.yml:9]: default test?

Comment (neillturner) [reply .github/workflows/backup-db.yml]: changed

File: .github/workflows/backup-db.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        type: choice
+        options:
+          - development
```
[the remark this answers is not in the document]
Comment (neillturner) [reply .github/workflows/backup-db.yml]: changed

**Review by RMcVelia — APPROVED** [review]

**Pull request merged**